### ISSUE TITLE
Display all video thumbnails in one frame in viewerType.scrollable

### ISSUE THREAD

**Issue by Helocominc** [sbis04/video_trimmer#181]
If the TrimViewer `type` is set to `ViewerType.scrollable` the video thumbnail doesn't display on a single frame  if `maxVideoLength` is provided. Only the provided `maxVideoLength` portion is shown in the thumbnail.

Platform: Android 
Version: video_trimmer 2.0.0

**Comment by sbis04** [comment]
Yeah, that's expected behavior. `maxVideoLength` specifies the maximum length the trimmed video can have. If you don't want to limit the length of the trimmed video, use `ViewerType.fixed`. Let me know if I didn't interpret your question properly.

**Comment by Helocominc** [comment]
I tried using `ViewerType.fixed` but the trimmer doesn't move over the thumbnails.

**Comment by sbis04** [comment]
Okay, got it. So, it's basically related to your previous issue (#178). Probably something like a scrollable trimmer without a max length limit should work then I guess?

**Comment by Helocominc** [comment]
I just want it to have a max length and then move. With all the thumbnail in one frame. Whatsapp have same functionality.

**Comment by Helocominc** [comment]
@sbis04  Any update based on my request?. It's important

**Comment by Helocominc** [comment]
@sbis04 please any progress for this issue ?
